wetterdienst/dwd/radar/sites.py

```diff
@@ -0,0 +1,135 @@
+from enum import Enum
+
+# https://docs.wradlib.org/en/stable/notebooks/radolan/radolan_network.html
+
+""" metadata about radar sites """
+RADAR_LOCATIONS = {
+    "asb": {
+        "site_name": "ASR Borkum",
+        "location_id": 10103,
+        "latitude": 53.564011,
+        "longitude": 6.748292,
+    },
+    "boo": {
+        "site_name": "Boostedt",
+        "location_id": 10132,
+        "latitude": 54.004381,
+        "longitude": 10.046899,
+    },
```

Comment:
Dear @meteoDaniel and @kmuehlbauer,

[wradlib's RADOLAN Radar Network documentation](https://docs.wradlib.org/en/stable/notebooks/radolan/radolan_network.html) has a `get_radar_locations()` function. I just recognized that the lat/lon values are slightly different at some places and digits. Also, `get_radar_locations()` features the altitude of the station.

So, I believe just using the `get_radar_locations()` function here 1:1 will be a better option as I believe you verified the positions of these radar sites at least twice before writing them down, @kmuehlbauer?

With kind regards,
Andreas.

Reply:
Only an alternative: We could parse the locations from here https://www.dwd.de/DE/derdwd/messnetz/atmosphaerenbeobachtung/_functions/HaeufigGesucht/koordinaten-radarverbund.pdf?__blob=publicationFile&v=5

Reply:
@amotl In `wradlib-notebooks` I've taken them from a predecessor of the paper @gutzbenj linked above. There's also a note in that paper that some coords changed slightly. Need to fix this in `wradlib-notebooks` too.

This is only needed to plot the radar circles on top of radolan for visualisation.

Reply:
> Need to fix this in wradlib-notebooks too.

I see, [1].

[1] https://github.com/wradlib/wradlib-notebooks/blob/v1.8.0/notebooks/radolan/radolan_network.ipynb

Reply:
With dd294dbe819251, I've added a parser for [1] and updated the `RADAR_LOCATIONS` dictionary appropriately with its outcome.

Everyone can run this process on their own behalf by invoking
```
python -m wetterdienst.dwd.radar.sites
```

[1] https://www.dwd.de/DE/derdwd/messnetz/atmosphaerenbeobachtung/_functions/HaeufigGesucht/koordinaten-radarverbund.pdf?__blob=publicationFile